code/modules/vehicles/vehicle_actions.dm

```diff
@@ -307,6 +307,14 @@
 			rider.Paralyze(40)
 			vehicle.visible_message(span_danger("[rider] misses the landing and falls on [rider.p_their()] face!"))
 		else
```

Comment:
A suggestion:
Rather then using an else here, and indenting further right, making things harder to read, why not early return? https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-early-returns

The same can actually be said of the if(.) at the top of this proc, I bet it'd make things a bit nicer to read